test/nn/test_convolution.py

```diff
@@ -1855,6 +1857,7 @@ def test_conv2d_same_padding_backward(self, device, dtype):
         self.assertEqual(gy_expect, y.grad)
 
     @dtypes(torch.double, torch.cdouble)
+    @skipMPS  # Double, complex double not supported on MPS
```

Comment:
IMO it's better this way, isn't it?
```suggestion
    @dtypesIfMPS(torch.float, torch.cfloat)  # Double, complex double not supported on MPS
```